Simplified/NYPLCatalogUngroupedFeedViewController.m

```diff
@@ -285,11 +285,14 @@ - (UIViewController *)previewingContext:(id<UIViewControllerPreviewing>)previewi
 {
   CGPoint referencePoint = [self.collectionView convertPoint:location fromView:self.view];
   NSIndexPath *indexPath = [self.collectionView indexPathForItemAtPoint:referencePoint];
-  NYPLBookNormalCell *cell = (NYPLBookNormalCell *)[self.collectionView cellForItemAtIndexPath:indexPath];
-  
+  UICollectionViewCell *cell = [self.collectionView cellForItemAtIndexPath:indexPath];
+  if (!cell || ![cell isKindOfClass:[NYPLBookNormalCell class]]) {
```

Comment:
This is a minor thing, but you just need `![cell isKindOfClass:[NYPLBookNormalCell class]]` here: The `!cell` check is redundant.